spring-boot-project/spring-boot-docs/src/docs/asciidoc/getting-started.adoc

```diff
@@ -346,7 +346,7 @@ If you need to solve a specific problem, check there first.
 
 You can shortcut the steps below by going to https://start.spring.io and choosing the "Web" starter from the dependencies searcher.
 Doing so generates a new project structure so that you can <<getting-started-first-application-code,start coding right away>>.
-Check the {spring-initializr-docs}/#user-guide[Spring Initializr documentation] for more details.
+Check the {spring-initializr-docs}[Spring Initializr documentation] for more details.
```

Comment:
I wasn't sure what more details the link to Initializr would give in that context, but I didn't want to remove it just yet. Just let me know and I'll remove it if you think it's not providing any benefit here.

Reply:
Yeah it's a good point. I wonder if https://github.com/spring-io/start.spring.io/blob/main/USING.adoc wouldn't be a better link now. What do you think?

Reply:
Seems like a better link - I'll adjust my PR.